API/Services/Tasks/Scanner/Parser/Parser.cs

```diff
@@ -25,15 +26,29 @@ public static class Parser
 
     public static readonly TimeSpan RegexTimeout = TimeSpan.FromMilliseconds(500);
 
-    public const string ImageFileExtensions = @"^(\.png|\.jpeg|\.jpg|\.webp|\.gif|\.avif)"; // Don't forget to update CoverChooser
+    public static Dictionary<string, List<string>> NonUniversalSupportedMimeMappings = new Dictionary<string, List<string>>()
```

Comment:
You need to leave some documentation about when to add certain formats to each section and how it works. For example UniversalFileImageExtension means all major browsers must support the format.

Reply:
Yes, to all that I didn't comment before. 

btw, what IDE are you using, can I get your styles also? Visual Studio is Rewriting the UTF-8 BOM every time it can... Also, I notice, some .cs has BOM, but they're a few that do not have.

Reply:
Edited on rider. 🤞